py/etrade-plan-pdf-tx-extract

```diff
@@ -39,22 +39,17 @@ class BenefitEntry:
    acquire_share_price: Decimal
    acquire_shares: int
 
-   sell_to_cover_tx_date: datetime.date
-   sell_to_cover_settle_date: datetime.date
-   sell_to_cover_price: Decimal
-   sell_to_cover_shares: int
-   sell_to_cover_fee: Decimal
+   sell_to_cover_tx_date: Optional[datetime.date]
+   sell_to_cover_settle_date: Optional[datetime.date]
+   sell_to_cover_price: Optional[Decimal]
+   sell_to_cover_shares: Optional[int]
+   sell_to_cover_fee: Optional[Decimal]
 
    plan_note: str
 
 def make_tx_renderer(benefits: List[BenefitEntry]):
    renderer = AcbCsvRenderer()
    for row, b in enumerate(benefits):
-      if b.sell_to_cover_tx_date is None or b.sell_to_cover_settle_date is None:
-         print("Error: No trade or settlement date found for sell-to-cover for "
-               f"{b.acquire_shares} shares of {b.security} aquired on {b.acquire_tx_date}")
-         exit(1)
```

Comment:
This error should probably be left in, though it can be constrained. It would make sense to fail if you detect any sell-to-cover attributes, but didn't find all of them. If the PDF format changes slightly and breaks a subset of these fields, it would be useful to know immediately.